test/test_html.rb

```diff
@@ -37,4 +37,10 @@ def test_works_with_multiple_children
 
     assert_equal "<ul><li>one</li><li>two</li></ul>", subject
   end
+
+  def test_works_with_void_tag
+    subject = ::Scarpe::HTML.render { |h| h.input(type: "text") }
```

Comment:
I think I copied this over here too, `HTML` would probably good enough

Reply:
well actually `Scarpe::HTML` in this scope I think

Reply:
In the tests, this doesn't work: `NameError: uninitialized constant TestHTML::HTML`